### PR TITLE
Remove old emergency banner

### PR DESCRIPTION
**Pull request by bevanloon**
Commit 905cea7f98f6ba4cb38695e054e691693f88e9ad added new code for a Redis powered emergency banner. The old emergency banner code and tests are no longer needed.

Trello card: https://trello.com/c/GZh4Rvpz/923-remove-old-emergency-banner-code-partials-and-tests

[Related PR 1217 in frontend](https://github.com/alphagov/frontend/pull/1217)
[Related PR 203 in govuk-developer-docs](https://github.com/alphagov/govuk-developer-docs/pull/203)
[Related PR 244 in fabric-scritps](https://github.com/alphagov/fabric-scripts/pull/244)

Once this is done, we should consider merging https://github.com/alphagov/static/pull/1001

### CODE REVIEW

File: app/views/root/_base.html.erb [hunk 1]
```diff
@@ -44,7 +39,7 @@
 
 <% content_for :content do %>
   <% unless @emergency_banner %>
-    <% show_global_bar = @banner_notification.blank? && !local_assigns[:hide_banner_notification] && !@emergency_banner %>
+    <% show_global_bar = !@emergency_banner %>
```
Comment (leenagupte) [inline app/views/root/_base.html.erb]: Do we still need to set `show_global_bar`?

Comment (bevanloon) [inline app/views/root/_base.html.erb]: good point....fixed that

**Comment by bevanloon** [comment]
@leenagupte think I've addressed everything with the latest push

File: app/views/root/_base.html.erb [hunk 2]
```diff
@@ -44,8 +39,7 @@
 
 <% content_for :content do %>
   <% unless @emergency_banner %>
-    <% show_global_bar = @banner_notification.blank? && !local_assigns[:hide_banner_notification] && !@emergency_banner %>
-    <%= render partial: "promo_banner" if show_global_bar %>
+    <%= render partial: "promo_banner" if !@emergency_banner %>
```
Comment (leenagupte) [inline app/views/root/_base.html.erb]: You don't need `if !@emergency_banner` here, it's handled by the `unless` in the previous line.

**Comment by leenagupte** [comment]
@bevanloon Commit structure looks good 👍 .

File: app/views/root/_base.html.erb (hunk 2 above)
Comment (bevanloon) [inline app/views/root/_base.html.erb]: @leenagupte fixed with 7959b60c5f40085a8b70be9646522fa8979dc60f

**Pull request merged**